tests/test_utils.py

```diff
@@ -246,30 +246,6 @@ def test_get_conda_envs_directory():
     assert get_conda_envs_directory(subprocess_output) == "/foo/bar/envs"
 
 
-# <<<<<<< HEAD
-# =======
-
-
-@pytest.mark.parametrize(
-    "argv,is_conda",
-    [
-        (["file", "conda"], False),
-        (["file", "batch"], False),
-        (["file", "conda:stuff"], False),
-        (["file", "batch:stuff"], False),
-        (["file", "--environment", "conda"], True),
-        (["file", "--environment", "other"], False),
-    ],
-)
-def test_is_mflow_conda_environment(argv, is_conda):
-    tmp = sys.argv.copy()
-    try:
-        sys.argv = argv
-        assert is_mflow_conda_environment() is is_conda
-    finally:
-        sys.argv = tmp
```

Comment:
This hasn't been deleted, it was added in #21 (just in a different part of the file)